src/charts/heatmap/heatmap-legend.component.js

```diff
@@ -0,0 +1,34 @@
+angular.module('patternfly.charts').component('pfHeatmapLegend', {
+  bindings: {
+    legend: '=?',
+    legendColors: '<?'
+  },
+  templateUrl: 'charts/heatmap/heatmap-legend.html',
+  controller: function () {
+    'use strict';
+    var ctrl = this;
+
+    var heatmapColorPatternDefaults = ['#d4f0fa', '#F9D67A', '#EC7A08', '#CE0000'];
+    var legendLabelDefaults = ['< 70%', '70-80%', '80-90%', '> 90%'];
+
+    ctrl.$onInit = function () {
+      var items = [];
+      var index;
+
+      //Allow overriding of defaults
+      if (!ctrl.legendColors) {
+        ctrl.legendColors = heatmapColorPatternDefaults;
+      }
+      if (!ctrl.legend) {
+        ctrl.legend = legendLabelDefaults;
+      }
+      for (index = ctrl.legend.length - 1; index >= 0; index--) {
+        items.push({
+          text: ctrl.legend[index],
+          color: ctrl.legendColors[index]
+        });
```

Comment:
I believe legendItems would need to be updated if ctrl.legend changes. Looks like an existing issue.